### PR TITLE
Fix indentation of courier doc comments

### PR DESCRIPTION
**Pull request by jpbetz**
See the forum question I has posted about the issue for details:

https://intellij-support.jetbrains.com/hc/en-us/community/posts/206536609-How-to-add-javadoc-style-documentation-comments-to-a-language-plugin-?page=1#community_comment_207121545

### CODE REVIEW

File: idea-plugin/META-INF/META-INF/plugin.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<idea-plugin version="2">
+  <id>com.your.company.unique.plugin.id</id>
```
Comment (saeta) [inline idea-plugin/META-INF/META-INF/plugin.xml]: Hey @jpbetz should this be filled in appropriately with `org.coursera...`?

File: idea-plugin/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<idea-plugin version="2">
+  <id>com.your.company.unique.plugin.id</id>
+  <name>Plugin display name here</name>
+  <version>1.0</version>
+  <vendor email="support@yourcompany.com" url="http://www.yourcompany.com">YourCompany</vendor>
```
Comment (saeta) [inline idea-plugin/resources/META-INF/plugin.xml]: Should these also be filled in appropriately?

How about:

```
<vendor email="courier@coursera.org" url="https://coursera.github.io/courier">Coursera</vendor>
```

File: idea-plugin/META-INF/META-INF/plugin.xml (hunk 1 above)
Comment (jpbetz) [inline idea-plugin/META-INF/META-INF/plugin.xml]: This file shouldn't exist.  I'm guessing that IntelliJ generated it automatically for some reason. I'll remove.

File: idea-plugin/resources/META-INF/plugin.xml (hunk 2 above)
Comment (jpbetz) [inline idea-plugin/resources/META-INF/plugin.xml]: Same with this file.  Looks like these files were generated when I was setting up a new IntelliJ install to build this plugin.

**Comment by saeta** [comment]
Excellent, this looks great. Thanks @jpbetz! I'm going to merge this and attempt to push a new release.

**Pull request merged**